### PR TITLE
Expanding on VReplication Migrations

### PR DESCRIPTION
**Pull request by FancyFane**
Signed-off-by: FancyFane <fane@planetscale.com>

Adding additional details and considerations when performing Migrations
Corrected the Workflow Show command, which had incorrect syntax. 

These changes are for v11 only, once feedback is provided, I will replicate these changes for v12 and v13.

### CODE REVIEW

File: content/en/docs/11.0/reference/vreplication/migrate.md [hunk 1]
```diff
@@ -14,62 +14,97 @@ This documentation is for a new (v2) set of vtctld commands that start in Vitess
...
 
 #### options
-<div class="cmd">
 
 Each `action` has additional options/parameters that can be used to modify its behavior.
 
-The options for the supported commands are the same as for `MoveTables`.
-
-</div>
+The options for the supported commands are the same as [MoveTables](../movetables), with the exception of `reverse_replication`. Please see "Differences between MoveTables and Migrate" below for an explanation.
```
Comment (hallaroo) [inline content/en/docs/11.0/reference/vreplication/migrate.md]: I'd add a link to the Differences section here.

File: content/en/docs/11.0/reference/vreplication/migrate.md [hunk 2]
```diff
@@ -14,62 +14,97 @@ This documentation is for a new (v2) set of vtctld commands that start in Vitess
...
-1. Confirm that data has been copied over correctly using [VDiff](../vdiff)
-1. Cleanup vreplication artifacts and source tables with [Complete](../complete) <br/>
+1. Initiate the migration using [Create](../create).<br/>
+`Migrate -all -source ext1.commerce Create commerce.wf`
+1. Monitor the workflow using [Show](../show).<br/>
+`Workflow commerce.wf Show`
+1. Confirm that data has been copied over correctly using [VDiff](../vdiff).<br/>
+`VDiff commerce.wf`
+1. Stop the application from writing to the source Vitess cluster. This is important as there is no reverse replication flow with Migrate.<br/>
+NOTE: Any writes to the source Vitess cluster performed after the migration completes will not be carried over to the target Vitess Cluster.
```
Comment (hallaroo) [inline content/en/docs/11.0/reference/vreplication/migrate.md]: Maybe add formatting to this note:
{{< info >}}
NOTE: Any writes to the source Vitess cluster performed after the migration completes will not be carried over to the target Vitess Cluster. 
{{< /info >}}

File: content/en/docs/11.0/reference/vreplication/migrate.md [hunk 3]
```diff
@@ -14,62 +14,97 @@ This documentation is for a new (v2) set of vtctld commands that start in Vitess
...
+
+### Network Considerations
+
+For Migrate to function properly, you will need to ensure communication is possible between the target Vitess cluster and the source Vitess cluster. At a minimum the following network concerns must be implemented:
+
+* Target vtctld/vttablet processes must reach the Source topo service.
+* Target vtctld/vttablet processes must reach EACH source vttablet's grpc port.
+    * You can limit your source vttablet's to just the replicas by using the `-tablet_types` option when creating the migration. 
+
+If you're migrating a keyspace from a production system, you may want to target a replica to reduce your load on the primary vtablets. This will also assist you in reducing the number of network considerations you need to make.
```
Comment (hallaroo) [inline content/en/docs/11.0/reference/vreplication/migrate.md]: If you're migrating a keyspace from a production system, you may want to target a replica to reduce your load on the primary vttablets.

**Review by hallaroo — CHANGES_REQUESTED** [review]
Couple suggestions and then I can approve

File: content/en/docs/11.0/reference/vreplication/migrate.md (hunk 1 above)
Comment (FancyFane) [reply content/en/docs/11.0/reference/vreplication/migrate.md]: I moved this closer to the top, and removed the "Please see below..." sentence.

**Review by hallaroo — APPROVED** [review]
LGTM!

**Pull request merged**